src/dataEdit/client.ts

```diff
@@ -80,9 +80,29 @@ function getPidFile(serverPort: number) {
 
 async function getOmegaEditPort() {
   if (omegaEditPort === 0) {
-    omegaEditPort = vscode.workspace
-      .getConfiguration('dataEditor')
-      .get<number>('omegaEditPort', DEFAULT_OMEGA_EDIT_PORT)
+    /**
```

Comment:
(the json configuration is using the literal property `dataEditor.omegaEditPort` (a property with a name that has a period in it), so this code doesn't work. I would recommend changing the configuration property to be hierarchical (no period) so that the current code will work.

Reply:
@arosien The current code does work. The reason

```ts
    omegaEditPort = vscode.workspace
      .getConfiguration('dataEditor')
      .get<number>('omegaEditPort', DEFAULT_OMEGA_EDIT_PORT)
```

didn't work is because it gets the default config setup in the `package.json` unless in debug mode. In debug mode it actually worked perfectly fine. The current update makes it work by looking at configs inside of `.vscode/launch.json`. The name of the variable wasn't the issue it was that the config desired wasn't being parsed yet.

Reply:
ah thanks, i misread the update of the property name in the new code!